### PR TITLE
Initial PR

### PR DESCRIPTION
**Pull request by HarithaIBM**
Initial PR.

### CODE REVIEW

**Review by MikeFultonDev — APPROVED** [review]
Nice!

**Comment by HarithaIBM** [comment]
> Nice!

Thank you, Mike!

File: buildenv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export ZOPEN_CHECK="${PWD}/unzip-tests/tests.sh"
+
+zopen_check_results()
+{
+  dir="$1"
+  pfx="$2"
+  chk="$1/$2_check.log"
+
+  # Echo the following information to guage build health
+  echo "actualFailures:0"
```
Comment (IgorTodorovskiIBM) [inline buildenv]: actualFailures is hardcoded to 0 here, same for the rest below

File: buildenv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+export ZOPEN_TYPE="GIT"
+export ZOPEN_CONFIGURE="cmake"
+export ZOPEN_CONFIGURE_OPTS=". -DBUILD_TESTING=ON -DENABLE_STATIC_INIT=ON -DZOSLIB_GENERIC=ON -DCMAKE_INSTALL_PREFIX=\$ZOPEN_INSTALL_DIR/ -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -D_XOPEN_SOURCE=600 "
+export ZOPEN_INSTALL="cmake"
+export ZOPEN_INSTALL_OPTS="--install . --prefix \$ZOPEN_INSTALL_DIR/"
+export ZOPEN_CHECK="${PWD}/unzip-tests/tests.sh"
+
+zopen_check_results()
+{
+  dir="$1"
```
Comment (IgorTodorovskiIBM) [inline buildenv]: dir and pfx is not used, you can remove it

File: buildenv (hunk 1 above)
Comment (HarithaIBM) [reply buildenv]: Corrected the script

File: buildenv (hunk 2 above)
Comment (HarithaIBM) [reply buildenv]: removed them!

**Comment by HarithaIBM** [comment]
Thank you for the review Mike and Igor!

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**